### DISCUSSION TITLE
Golang version management

### DISCUSSION THREAD

**Discussion by StaphSynth** [renovatebot/renovate#18415] [Request Help]
Hi, I maintain a repository using Renovate and it keeps opening PRs trying to bump the Golang version. This would be fine except it's not bumping it in all the required files, it does not update the `GO_VERSION` env var specified in our Github actions yml file, which means that automated tests and builds end up running against a different version to dev and production.

I've been through the docs, but I still have no idea how to tell renovate to also include this file. Could someone help, please?

**Comment by viceice** [comment]
you need to configure a regex manager
